Doc/library/urllib.parse.rst

```diff
@@ -451,13 +451,16 @@ task isn't already covered by the URL parsing functions above.
 .. function:: quote(string, safe='/', encoding=None, errors=None)
 
    Replace special characters in *string* using the ``%xx`` escape. Letters,
-   digits, and the characters ``'_.-'`` are never quoted. By default, this
+   digits, and the characters ``'_.-~'`` are never quoted. By default, this
    function is intended for quoting the path section of URL. The optional *safe*
    parameter specifies additional ASCII characters that should not be quoted
    --- its default value is ``'/'``.
 
    *string* may be either a :class:`str` or a :class:`bytes`.
 
+   Python 3.7 updates from using RFC 2396 to RFC 3986 to quote URL strings.
+   Now, "~" is included in the set of reserved characters.
```

Comment:
This should use a `.. versionchanged: 3.7` note (there are several examples for that earlier in the file).